cmp/grid/columns/Column.js

```diff
@@ -532,6 +536,16 @@ export class Column {
                         agNodeB
                     };
 
+                if (isString(sortValue)) {
+                    // If sortValue points to a different field
+                    valueA = recordA?.data[sortValue] ?? valueA; // Group rows have no record.
```

Comment:
this note about groupRows seems problematic?  So will this even work or a groupedGrid that is trying to use the new sortValue property?

Reply:
Well the value for the grouped row was being passed to the comparator before regardless. It was an empty string. Which it remains here.

Reply:
Tested whilst paying attn to the group rows. The sort works, but the group rows now jump around with the when sorting by this column. Previously the groups would stay put. Will revisit.